src/next.config.js

```diff
@@ -58,4 +58,7 @@ module.exports = withCSS({
 
     return config;
   },
+  publicRuntimeConfig: {
+    openCollectiveRedirectUrl: process.env.OPENCOLLECTIVE_REDIRECT_URL,
```

Comment:
This might be a valid solution, but we should not have 2 different ways of doing the same thing. 

So please process `OPENCOLLECTIVE_REDIRECT_URL` like `OPENCOLLECTIVE_REFERRAL`